src/helper/serve.utils.ts

```diff
@@ -102,12 +104,41 @@ export async function startServer({ domain, host, isSSR, serverPort }) {
 		res.json({ ok: 'ok' });
 	});
 
+	// parse application/x-www-form-urlencoded
+	app.use(express.json());
+
 	// app.use('/platform', proxy(`https://${host}`, {
 	//   proxyReqPathResolver: function (req) {
 	//     return `/platform${req.url}`;
 	//   }
 	// }));
-	app.use(`/api`, proxy(`${host}`));
+	  
+	const options = {
+		target: host, // target host
+		changeOrigin: true, // needed for virtual hosted sites
+		cookieDomainRewrite: 'localhost', // rewrite cookies to localhost
+		onProxyReq: fixRequestBody, //to fix body parser issue  
+	  };
+
+	  // proxy to solve CORS issue
+	  const corsProxy = createProxyMiddleware(options);
+	  app.use('/service', async (req,res,next) => {
```

Comment:
Shouldn't we add `/ext` as well here? It will effects themes like tira which are using data loaders.